README.md

```diff
@@ -40,7 +40,7 @@ from darts import TimeSeries
 
 df = pd.read_csv('AirPassengers.csv', delimiter=",")
 series = TimeSeries.from_dataframe(df, 'Month', '#Passengers')
-train, val = series.split_after(pd.Timestamp('19590101'))
+train, val = series.split_after(pd.Timestamp('19580101'))
```

Comment:
Doesn't seem to be related to this PR

Reply:
It's not really, but since this PR was small and already about updating the README I thought I could change the example image in the same PR instead of opening a new one…